src/integration/wrapper.go

```diff
@@ -69,7 +69,12 @@ func WithHTTPSink(sinkConfig config.HTTPSink) OptionFunc {
 			}
 		}
 
-		prober, err := prober.New(iw.probeTimeout, iw.probeBackoff, prober.WithLogger(iw.logger), prober.WithClient(client))
+		probeTimeout := sinkConfig.ProbeTimeout
+		// if the ProbeTimeout is not set in HTTPSink, just use the default probe timeout from the wrapper
+		if probeTimeout == 0*time.Second {
```

Comment:
You can use `time.IsZero()` here.

Reply:
probeTimeout is of time.Duration. I cannot use time.IsZero() here.
I want to check whether it is set or not here. If not set, then probeTimeout should be zero duration

Reply:
you can probably use `0` on the RHS. instead of using `*time.Second`
The compiler should use the LHS type to automatically use a time.Duration typed zero value.

Reply:
done